src/super_gradients/training/models/detection_models/pp_yolo_e/pan.py

```diff
@@ -0,0 +1,185 @@
+import collections
+from typing import Type, Tuple, List
+
+import torch
+from super_gradients.common.decorators.factory_decorator import resolve_param
+from super_gradients.common.factories.activations_type_factory import ActivationsTypeFactory
+from torch import nn, Tensor
+from super_gradients.training.models.detection_models.csp_resnet import CSPResNetBasicBlock
+from super_gradients.modules import ConvBNAct
+
+__all__ = ["CustomCSPPAN"]
+
+
+class SPP(nn.Module):
```

Comment:
Yes, these two module looks similar. But they have different name of internal layers. 
This would require us to update pretrained checkpoints on S3 for ppyolo model or retrain them.

SPP block from YoloX could also be refactored to use `ConvBNAct` module, but that would touch YoloX checkpoints as well. I'd rather not open this can of worms for this task, tbh.